### PR TITLE
Rename to participant, add rootNavigator to TabNavigator

### PR DESCRIPTION
**Pull request by dsznajder**

### CODE REVIEW

File: src/scenes/Comments/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    flexWrap: 'wrap',
+    justifyContent: 'space-between',
+    margin: 10,
+  },
+  header: {
+    color: PRIMARY_COLOR,
+    fontSize: 26,
+  },
+  section: {
+    color: PRIMARY_COLOR,
```
Comment (Darex1991) [inline src/scenes/Comments/index.js]: `PrimaryColor`

File: src/scenes/Login/index.js [hunk 2]
```diff
@@ -57,7 +44,9 @@ export default class Login extends Component {
 
 const styles = StyleSheet.create({
   container: {
+    flex: 1,
     alignItems: 'center',
```
Comment (Darex1991) [inline src/scenes/Login/index.js]: alphabetic

File: src/scenes/Organization/OrganizationForm.js [hunk 3]
```diff
@@ -26,21 +27,35 @@ export class OrganizationForm extends Component {
...
-        </FormValidationMessage>
+      <View style={styles.container}>
+        <View style={styles.form}>
+          <FormLabel>Name</FormLabel>
+          <FormInput
+            onChangeText={value => this._onChange('name', value)}
+            ref={ref => (this.nameInput = ref)}
+          />
+          <FormValidationMessage>
+            {this.state.error && <Text>Name cannot be blank</Text>}
```
Comment (Darex1991) [inline src/scenes/Organization/OrganizationForm.js]: I18n

File: src/scenes/Organization/OrganizationForm.js [hunk 4]
```diff
@@ -26,21 +27,35 @@ export class OrganizationForm extends Component {
...
         <Button onPress={this._onSubmit} title="Create Organization" />
       </View>
     )
   }
 }
 
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
```
Comment (Darex1991) [inline src/scenes/Organization/OrganizationForm.js]: alphabetic

File: src/components/TopBar.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ export class TopBar extends Component {
 
       return (
         <Header
-          centerComponent={{ text: 'Merit Money', style: { color: 'white', fontSize: 24 } }}
+          centerComponent={{ text: 'MeritMoney', style: { color: 'white', fontSize: 24 } }}
```
Comment (Darex1991) [inline src/components/TopBar.js]: without inline styles

File: src/scenes/Comments/index.js (hunk 1 above)
Comment (dsznajder) [reply src/scenes/Comments/index.js]: I'm not sure about that one, You can misunderstood this as a component ;)

File: src/components/TopBar.js (hunk 5 above)
Comment (dsznajder) [reply src/components/TopBar.js]: :+1:

File: src/scenes/Organization/OrganizationForm.js (hunk 3 above)
Comment (dsznajder) [reply src/scenes/Organization/OrganizationForm.js]: We are not creating currently a international version. Conclusion -> We only support english :)

**Pull request merged**